rpm/SPECS/pepProxy.spec

```diff
@@ -90,18 +90,6 @@ echo "[INFO] Configuring application"
     setfacl -d -m g::rwx %{_pepProxy_log_dir}
     setfacl -d -m o::rx %{_pepProxy_log_dir}
 
-    echo "[INFO] Checking Context Broker installations"
```

Comment:
CHANGES_NEXT_RELEASE entry needed? It is a change in the way the RPM behaves, it makes sense to me mention it in the changelog buffer file.